tests/loggers/test_neptune.py

```diff
@@ -276,14 +276,15 @@ def test_after_save_checkpoint(self, neptune):
             logger, run_instance_mock, run_attr_mock = self._get_logger_with_mocks(
                 api_key="test", project="project", **prefix
             )
+            models_root_dir = os.path.join("path", "to", "models")
             cb_mock = MagicMock(
-                dirpath="path/to/models",
-                last_model_path="path/to/models/last",
+                dirpath=models_root_dir,
+                last_model_path=os.path.join(models_root_dir, "last"),
                 best_k_models={
-                    "path/to/models/model1": None,
-                    "path/to/models/model2/with/slashes": None,
+                    f"{os.path.join(models_root_dir, 'model1')}": None,
+                    f"{os.path.join(models_root_dir, 'model2/with/slashes')}": None,
```

Comment:
maybe still replace / by path join here

Reply:
By `model2/with/slashes` we mean checkpoints produced via (we already saw a few examples of it):
```
ModelCheckpoint(dirpath="model/checkpoints/",
                               filename="{epoch:02d}-{val/loss:.2f}",
                               save_weights_only=True,
                               save_top_k=3,
                               save_last=True,
                               monitor="val/loss",
                               every_n_epochs=1)
```
which produces checkpoints like `epoch=00-val/loss=0.16.ckpt`. So it's with a "/" slash symbol in the filename and I don't know how exactly Windows handle it :wink: and that's the reason that it wasn't joined via `os.path.join`.

Check these as an reference: https://app.neptune.ai/o/common/org/pytorch-lightning-integration/e/PTL-1/all?path=training%2Fmodel%2Fcheckpoints%2Fepoch%3D00-val